recipes/tinyexr/all/patches/0001-use-conan-deps.patch

```diff
@@ -0,0 +1,66 @@
+--- tinyexr.h
++++ tinyexr.h
+@@ -547,10 +547,11 @@
+ #endif
+ 
+ #if TINYEXR_USE_MINIZ
++#include <miniz.h>
+ #else
+ //  Issue #46. Please include your own zlib-compatible API header before
+ //  including `tinyexr.h`
+-//#include "zlib.h"
++#include "zlib.h"
+ #endif
+ 
+ #if TINYEXR_USE_ZFP
+@@ -588,6 +589,7 @@
+ #endif
+ #endif
+ 
++#if 0
+ #if TINYEXR_USE_MINIZ
+ 
+ namespace miniz {
+@@ -7048,6 +7050,7 @@
+ #endif
+ 
+ #endif  // TINYEXR_USE_MINIZ
++#endif  // 0
+ 
+ // static bool IsBigEndian(void) {
+ //  union {
+@@ -7650,11 +7653,11 @@
+   // Compress the data using miniz
+   //
+ 
+-  miniz::mz_ulong outSize = miniz::mz_compressBound(src_size);
+-  int ret = miniz::mz_compress(
++  mz_ulong outSize = mz_compressBound(src_size);
```

Comment:
why namespace was changed?

Reply:
Embedded miniz was surrounded with `miniz` namespace and original miniz doesn't use any namespace. I changed tinyexr to use miniz from CCI.